### PR TITLE
JSO @ JuliaCon 2023

### PR DESCRIPTION
**Pull request by abelsiqueira**
@tmigot, I've added your talk with a short news. Feel free to extend it, if you want.

### CODE REVIEW

**Comment by tmigot** [comment]
Thanks @abelsiqueira for initiating this. I try to expand it a bit. Can you have a look if that's ok with you?

File: news-and-blogposts/2023/2023-09-27-juliacon-2023-talk.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@def title = "Julia Smooth Optimizers @ JuliaCon 2023"
+@def rss_description = "The JuliaCon 2023 talk about Julia Smooth Optimizers is up on YouTube. Check the latest updates of the packages given by Tangi Migot."
+
+# Julia Smooth Optimizers @ JuliaCon 2023
+
+This year JuliaCon took place at MIT, Cambridge, USA.
+This year’s conference was particularly special, as it marked our first in-person attendance after [the previous year’s online edition](https://jso.dev/news-and-blogposts/2022/2022-07-30-juliacon22/).
+[Tangi Migot](https://tmigot.github.io) gave a talk titled “Optimization solvers in JuliaSmoothOptimizers” during the [JuMP-dev](https://jump.dev/meetings/jumpdev2023/) track.
```
Comment (abelsiqueira) [inline news-and-blogposts/2023/2023-09-27-juliacon-2023-talk.md:8]:
```suggestion
This year JuliaCon took place at MIT, Cambridge, USA.
This year's conference was particularly special, as it marked our first in-person attendance after [the previous year’s online edition](https://jso.dev/news-and-blogposts/2022/2022-07-30-juliacon22/).
[Tangi Migot](https://tmigot.github.io) gave a talk titled "Optimization solvers in JuliaSmoothOptimizers" during the [JuMP-dev](https://jump.dev/meetings/jumpdev2023/) track.
```

**Comment by abelsiqueira** [comment]
Thanks, this looks great. We can add that other talk into our references as well, if you want to do it.

**Comment by tmigot** [comment]
> Thanks, this looks great. We can add that other talk into our references as well, if you want to do it.

Good suggestion, it is done.

**Review by tmigot — APPROVED** [review]

**Comment by abelsiqueira** [comment]
Thanks!

**Pull request merged**